vmdb/config/initializers/fast_gettext.rb

```diff
@@ -18,17 +18,27 @@ def fix_i18n_available_locales
   I18n.available_locales += FastGettext.available_locales.grep(/_/).map { |i| i.gsub("_", "-") }
 end
 
-locale_path = Rails.root.join("config/locales")
+old_debug, $DEBUG = $DEBUG, nil
```

Comment:
@matthewd is `silence` or `silence_stream` useable here?  http://apidock.com/rails/Kernel/silence_stream?